docs/src/main/asciidoc/security-overview-concept.adoc

```diff
@@ -2,68 +2,67 @@
 
-Before you start building security into your Quarkus applications, learn about the xref:security-architecture-concept.adoc[Quarkus Security architecture] and the different authentication mechanisms and features that you can use.
+Before building security into your Quarkus applications, learn about the xref:security-architecture-concept.adoc[Quarkus Security architecture] and the different authentication mechanisms and features that you can use.
 
 == Key features of Quarkus Security
 
-The Quarkus Security framework provides built-in security authentication mechanisms for Basic, Form-based, and mutual TLS (mTLS) authentication. 
-You can also use other well-known xref:security-authentication-mechanisms-concept.adoc#other-supported-authentication-mechanisms[authentication mechanisms], such as OpenId Connect (OIDC) and WebAuthn.
+The Quarkus Security framework provides built-in security authentication mechanisms for Basic, Form-based, and mutual TLS (mTLS) authentication.
+You can also use other well-known xref:security-authentication-mechanisms-concept.adoc#other-supported-authentication-mechanisms[authentication mechanisms], such as OpenID Connect (OIDC) and WebAuthn.
 
 Authentication mechanisms depend on xref:security-identity-providers-concept.adoc[Identity providers] to verify the authentication credentials and map them to a `SecurityIdentity` instance, which has the username, roles, original authentication credentials, and other attributes.
 
-{project-name} also includes built-in security to allow for role-based access control (RBAC) based on the common security annotations @RolesAllowed, @DenyAll, @PermitAll on REST endpoints, and CDI beans. 
-For more information, see xref:security-authorize-web-endpoints-reference.adoc[Authorization of web endpoints].
+{project-name} also includes built-in security to allow for role-based access control (RBAC) based on the common security annotations @RolesAllowed, @DenyAll, @PermitAll on REST endpoints, and CDI beans.
+For more information, see the Quarkus xref:security-authorize-web-endpoints-reference.adoc[Authorization of web endpoints] guide.
 
 Quarkus Security also supports the following features:
 
 * xref:security-proactive-authentication-concept.adoc[Proactive authentication]
 * xref:http-reference.adoc#ssl[Secure connections with SSL/TLS]
-* <<cross-origin-resource-sharing>>
-* <<csrf-prevention>>
-* <<samesite-cookies>>
-* <<secrets-engines>>
-* <<secure-serialization>>
-* <<rest-data-panache>>
+* xref:cross-origin-resource-sharing[Cross-origin resource sharing]
+* xref:crsf-prevention[Cross-site Request Forgery (CSRF) prevention]
+* xref:samesite-cookies[SameSite cookies]
+* xref:secrets-engines[Secrets engines]
+* xref:secure-resources-by-rest-with-panache[Secure auto-generated resources by REST Data with Panache]
+* xref:secure-serialization[Secure serialization]
```

Comment:
`<<>>` was replaced by `xref` in https://github.com/quarkusio/quarkus/pull/34070 too

These changes should be also reverted if this gets reverted @michelle-purcell / @sberyozkin

Reply:
Yeah we had this discussion and we decided to stick to `<<>>` to avoid duplications and I worked hard on making sure things would work in downstream documentation thanks to a transformer.

Reply:
@rsvoboda - Thanks for flagging this :+1: 

Apologies for the churn. I did mean to come back and tidy these up as per the recent updates that @MichalMaler  added to the [Quarkus contributor guide](https://quarkus.io/guides/doc-reference#cross-references).

I have now fixed these anchors by using <<section-id, human readable title>>. Naked anchors without the title do not render downstream on PV2.

xrefs (my previous commit) and <<section-id, human readable title>> both render OK on our downstream PV2 system and are both valid AsciiDoc elements.  I verified on our PV2 sandbox.

@gsmet - Thanks so much for helping to make our lives easier downstream :green_heart: 

We really appreciate your help and efforts to reduce the manual workload to make things work downstream without the massive manual editing that we are forced to do today. However, we were unaware that this transform piece to fix our problem with naked anchors <<>> was finished. 
Can you confirm where in the repo does the transformer recreate the 'clean' asciidoc that we can consume? A quick demo and some clarification would be helpful.

FYI we are currently (manually :cry: )  importing the raw adoc upstream doc source adoc file and all of its dependants includes::, images, etc directly from https://github.com/quarkusio/quarkus/tree/<applicable-branch>/docs/src/main/asciidoc, which is the source state before any transforming is done.   

Thanks